.istanbul.yml

```diff
@@ -1,6 +1,6 @@
 instrumentation:
   include-all-sources: true
-  excludes: ['**/migrations/**', '**/deploy/**', '**/src/domain/settlements/settlementEventListener.js']
+  excludes: ['**/migrations/**', '**/deploy/**', '**/src/domain/settlements/settlementEventListener.js', '**/src/handlers/lib/kafka/testProducer.js', '**/testPI2/**']
```

Comment:
testProducer.js should go under the test folder.

Reply:
changed

Reply:
@rajivmothilal I don't see that change reflected here?